docs/source/developers/cpp/building.rst

```diff
@@ -135,7 +135,7 @@ Minimal release build:
    cmake ..
    make
 
-Minimal debug build with unit tests:
+Minimal debug build with unit tests (4Gb of RAM for building or more recommended):
```

Comment:
I think you are referring to bytes (GB) and not bits (Gb).

Reply:
Thanks.